uk.co.compucorp.civicrm.tasksassignments/js/test/crm-tasks-workflows/globals.js

```diff
@@ -0,0 +1,6 @@
+/* eslint no-unused-vars: 0 */
+'use strict';
+
+var CRM = {
+  vars: {}
```

Comment:
You can put the `'caseType.category': 80` part here, then you dont need to mock it in spec files.

Reply:
I can add it here, but I rather mock it because mocking it means I don't need to understand where the value is coming from. I think that's the purpose of providing it in a constant.